examples/evolve_dstream_surf.py

```diff
@@ -47,9 +67,19 @@ def evolve_drift(population: typing.List) -> typing.List:
             parent.CreateOffspring()
             for parent in selector.choices(population[:nsplit], k=nsplit)
         ]
+        if fossil_interval and generation % fossil_interval == 0:
+            fossils.extend(
+                [
+                    parent.CreateOffspring()
```

Comment:
We should briefly note why we’re doing the CreateOffspring trick (otherwise this is totally something I’d forget)